### DISCUSSION TITLE
latent generations in serpentTools.SensitivityReader

### DISCUSSION THREAD

**Discussion by rababahm** [CORE-GATECH-GROUP/serpent-tools#476] [Q&A] [answered]
Hello,
Does the serpentTools.SensitivityReader has an option to read sensitivities for different latnet genertions number? and plot the sensitivity with spcicfic latent generations number?

Thank you!

**Comment by drewejohnson** [comment, marked as answer]
Hey @rababahm! Sorry for the very late delay on this. And welcome to the discussion forum!

We do not yet support parsing arrays from latent generations in the sensitivity reader - #367. If this is a vital feature, please make some comments there and we can think about a solution
